### PR TITLE
Add mailer functionality

### LINKED ISSUE #25: Add email verification functionality
Implement email verification functionality.

### PR DESCRIPTION
**Pull request by ZuluNovember**
closes #25,

# Add mailer functionality

- Added nodemailer and setup transporter

### Example usage
```
const sendMail = require(pathToMailer);

sendMail(targetAdress, subject, html);  
```

### CODE REVIEW

**Review by Kishimoto96 — APPROVED** [review]
Looks good! super clear

File: src/utils/mailer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  secure: false,
+  auth: {
+    user: process.env.GOOGLE_ACCOUNT,
+    pass: process.env.GOOGLE_PASSWORD,
+  },
+});
+
+const sendMail = async (targetMail, subject, html) => {
+  const info = await transporter.sendMail({
+    from: process.env.GOOGLE_ACCOUNT,
```
Comment (Shrreya) [inline src/utils/mailer.js:15]: I think Nodemailer allows you to format the sender address as : `'"Display Name" <foo@example.com>'`
So you can leverage that feature and add your display name as something like "Team Voluntera".
Then your automated emails will look more professional.

(Extra tip - Add a cool profile picture like a company logo on the email ID if you haven't already. Then receivers will see a sender display name as well as display picture.)

**Review by Shrreya — COMMENTED** [review]
@ZuluNovember Looks good! I just have one small suggestion below.

Also a reminder - Once the controller that calls this util function is written, make sure it is tested using mocking.

**Comment by Shrreya** [comment]
@ZuluNovember Can you merge main branch and resolve merge conflicts?

**Comment by ZuluNovember** [comment]
> @ZuluNovember Can you merge main branch and resolve merge conflicts?

@Shrreya Sorry I should've done it. Now it's done.

**Review by Shrreya — APPROVED** [review]

**Pull request merged**